src/model/data.types.model.ts

```diff
@@ -22,7 +22,11 @@ export enum NatureOfControlType {
   over_25_percent_of_shares = "over_25_percent_of_shares",
   over_25_percent_of_voting_rights = "over_25_percent_of_voting_rights",
   appoint_or_remove_majority_board_directors = "appoint_or_remove_majority_board_directors",
-  significant_influence_or_control = "significant_influence_or_control"
+  significant_influence_or_control = "significant_influence_or_control",
+  shares = "shares",
```

Comment:
Should already been covered, don't you think?

Reply:
yes the existing 
  over_25_percent_of_shares = "over_25_percent_of_shares",
  over_25_percent_of_voting_rights = "over_25_percent_of_voting_rights",
  appoint_or_remove_majority_board_directors = "appoint_or_remove_majority_board_directors",
  significant_influence_or_control = "significant_influence_or_control",
should cover the 4 options in each NOC type

Reply:
I would describe these as "verbose". I was attempting to simply the enum names, I thought I'd take the opportunity as we are changing both the web and api. If we are happy to continue with them then I will change this.

Reply:
yes I would stick with the api naming as we have spent time modelling it so that it has consistent naming across the BOs and MOs. So stick to the api naming where possible, if there are new fields being added in the v5 page that don't line up with what we have in the api spec then we will add them to the api spec in the subtask. The api subtask is not meant to re-model the api spec, but to add any extra fields that were introduced in the v5 page. Any new fields added should also be snake case.